### ISSUE TITLE
[QA] Flaky accessibility tests

### ISSUE THREAD

**Issue by saw-jan** [owncloud/ocis#11847]
Some web e2e tests are failing due to accessibility violations:
- https://drone.owncloud.com/owncloud/ocis/49433/43/10
- https://drone.owncloud.com/owncloud/ocis/49402/47/15

```feature
Scenario: Alice can share this weeks meal plan with all parents # tests/e2e/cucumber/features/journeys/kindergarten.feature:21
(node:66) Warning: Setting the NODE_TLS_REJECT_UNAUTHORIZED environment variable to '0' makes TLS connections and HTTPS requests insecure by disabling certificate verification.
(Use `node --trace-warnings ...` to show where the warning was created)
    Given "Admin" creates following users using API
      │ id    │
      │ Alice │
      │ Brian │
      │ Carol │
    And "Admin" creates following group using API
      │ id       │
      │ sales    │
      │ security │
    And "Admin" adds user to the group using API
      │ user  │ group │
      │ Brian │ sales │
    When "Alice" logs in
    And "Alice" navigates to the personal space page
    And "Alice" creates the following resources
      │ resource                             │ type   │
      │ groups/Kindergarten Koalas/meal plan │ folder │
      │ groups/Pre-Schools Pirates/meal plan │ folder │
      │ groups/Teddy Bear Daycare/meal plan  │ folder │
    ✖ failed
      Error: expect(received).toHaveLength(expected)

      Expected length: 0
      Received length: 2
      Received array:  [{"description": "Ensure every ARIA input field has an accessible name", "help": "ARIA input fields must have an accessible name", "helpUrl": "https://dequeuniversity.com/rules/axe/4.11/aria-input-field-name?application=playwright", "id": "aria-input-field-name", "impact": "serious", "nodes": [{"all": [], "any": [{"data": null, "id": "aria-label", "impact": "serious", "message": "aria-label attribute does not exist or is empty", "relatedNodes": []}, {"data": null, "id": "aria-labelledby", "impact": "serious", "message": "aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty", "relatedNodes": []}, {"data": {"messageKey": "noAttr"}, "id": "non-empty-title", "impact": "serious", "message": "Element has no title attribute", "relatedNodes": []}], "failureSummary": "Fix any of the following:
        aria-label attribute does not exist or is empty
        aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
        Element has no title attribute", "html": "<div class=\"tippy-box\" data-state=\"hidden\" tabindex=\"-1\" data-theme=\"none\" data-animation=\"fade\" role=\"listbox\" data-placement=\"bottom-start\" style=\"max-width: 416px; transition-duration: 250ms;\">", "impact": "serious", "none": [], "target": [".tippy-box"]}], "tags": ["cat.aria", "wcag2a", "wcag412", "TTv5", "TT5.c", "EN-301-549", "EN-9.4.1.2", "ACT", "RGAAv4", "RGAA-11.1.1"]}, {"description": "Ensure elements with an ARIA role that require child roles contain them", "help": "Certain ARIA roles must contain particular children", "helpUrl": "https://dequeuniversity.com/rules/axe/4.11/aria-required-children?application=playwright", "id": "aria-required-children", "impact": "critical", "nodes": [{"all": [], "any": [{"data": {"messageKey": "unallowed", "values": "ul[tabindex]"}, "id": "aria-required-children", "impact": "critical", "message": "Element has children which are not allowed: ul[tabindex]", "relatedNodes": [{"html": "<ul class=\"oc-list oc-my-rm oc-mx-rm expanded-list\" id=\"create-list\">", "target": ["#create-list"]}, {"html": "<ul class=\"oc-list oc-my-rm oc-mx-rm\">", "target": ["ul:nth-child(2)"]}, {"html": "<ul class=\"oc-list oc-my-rm oc-mx-rm\">", "target": ["ul:nth-child(3)"]}]}], "failureSummary": "Fix any of the following:
        Element has children which are not allowed: ul[tabindex]", "html": "<div class=\"tippy-box\" data-state=\"hidden\" tabindex=\"-1\" data-theme=\"none\" data-animation=\"fade\" role=\"listbox\" data-placement=\"bottom-start\" style=\"max-width: 416px; transition-duration: 250ms;\">", "impact": "critical", "none": [], "target": [".tippy-box"]}], "tags": ["cat.aria", "wcag2a", "wcag131", "EN-301-549", "EN-9.1.3.1", "RGAAv4", "RGAA-9.3.1"]}]
          at Proxy.<anonymous> (/drone/src/webTestRunner/node_modules/.pnpm/playwright@1.56.1/node_modules/playwright/lib/matchers/expect.js:221:24)
          at createNewFolder (file:///drone/src/webTestRunner/tests/e2e/support/objects/app-files/resource/actions.ts:211:113)
          at async Module.createResources (file:///drone/src/webTestRunner/tests/e2e/support/objects/app-files/resource/actions.ts:375:13)
          at async Resource.create (file:///drone/src/webTestRunner/tests/e2e/support/objects/app-files/resource/index.ts:10:9)
          at async World.<anonymous> (file:///drone/src/webTestRunner/tests/e2e/cucumber/steps/ui/resources.ts:12:9)
```

And on second try:

```feature
  Scenario: Alice can share this weeks meal plan with all parents # tests/e2e/cucumber/features/journeys/kindergarten.feature:21
    Given "Admin" creates following users using API
      │ id    │
      │ Alice │
      │ Brian │
      │ Carol │
    And "Admin" creates following group using API
      │ id       │
      │ sales    │
      │ security │
    And "Admin" adds user to the group using API
      │ user  │ group │
      │ Brian │ sales │
    When "Alice" logs in
    And "Alice" navigates to the personal space page
    And "Alice" creates the following resources
      │ resource                             │ type   │
      │ groups/Kindergarten Koalas/meal plan │ folder │
      │ groups/Pre-Schools Pirates/meal plan │ folder │
      │ groups/Teddy Bear Daycare/meal plan  │ folder │
    And "Alice" uploads the following resources
      │ resource          │ to                                   │
      │ PARENT/parent.txt │ groups/Kindergarten Koalas/meal plan │
      │ lorem.txt         │ groups/Kindergarten Koalas/meal plan │
      │ lorem-big.txt     │ groups/Kindergarten Koalas/meal plan │
      │ data.zip          │ groups/Pre-Schools Pirates/meal plan │
      │ lorem.txt         │ groups/Pre-Schools Pirates/meal plan │
      │ lorem-big.txt     │ groups/Pre-Schools Pirates/meal plan │
      │ data.zip          │ groups/Teddy Bear Daycare/meal plan  │
      │ lorem.txt         │ groups/Teddy Bear Daycare/meal plan  │
      │ lorem-big.txt     │ groups/Teddy Bear Daycare/meal plan  │
    And "Alice" shares the following resources using the sidebar panel
      │ resource                                           │ recipient │ type  │ role                      │ resourceType │
      │ groups/Pre-Schools Pirates/meal plan               │ Brian     │ user  │ Can edit without versions │ folder       │
      │ groups/Pre-Schools Pirates/meal plan               │ Carol     │ user  │ Can edit without versions │ folder       │
      │ groups/Pre-Schools Pirates/meal plan/lorem-big.txt │ sales     │ group │ Can view                  │ file         │
      │ groups/Pre-Schools Pirates/meal plan/lorem-big.txt │ Carol     │ user  │ Can view                  │ file         │
      │ groups/Kindergarten Koalas/meal plan               │ sales     │ group │ Can view                  │ folder       │
      │ groups/Kindergarten Koalas/meal plan               │ security  │ group │ Can edit without versions │ folder       │
      │ groups/Kindergarten Koalas/meal plan/lorem.txt     │ sales     │ group │ Can view                  │ file         │
      │ groups/Kindergarten Koalas/meal plan/lorem.txt     │ security  │ group │ Can view                  │ file         │
      │ groups/Teddy Bear Daycare/meal plan                │ Brian     │ user  │ Can edit without versions │ folder       │
      │ groups/Teddy Bear Daycare/meal plan                │ Carol     │ user  │ Can edit without versions │ folder       │
      │ groups/Teddy Bear Daycare/meal plan/data.zip       │ Brian     │ user  │ Can edit without versions │ file         │
      │ groups/Teddy Bear Daycare/meal plan/data.zip       │ Carol     │ user  │ Can edit without versions │ file         │
    And "Alice" updates following sharee role
      │ resource                                           │ recipient │ type  │ role                      │ resourceType │
      │ groups/Pre-Schools Pirates/meal plan               │ Carol     │ user  │ Can view                  │ folder       │
      │ groups/Pre-Schools Pirates/meal plan/lorem-big.txt │ sales     │ group │ Can edit without versions │ file         │
      │ groups/Kindergarten Koalas/meal plan               │ sales     │ group │ Can edit without versions │ folder       │
      │ groups/Teddy Bear Daycare/meal plan/data.zip       │ Carol     │ user  │ Can edit without versions │ file         │
    When "Brian" logs in
    And "Brian" navigates to the shared with me page
    And "Brian" downloads the following resources using the sidebar panel
      │ resource │ from      │ type │
      │ data.zip │ meal plan │ file │
    When "Carol" logs in
    And "Carol" navigates to the shared with me page
    And "Carol" downloads the following resources using the sidebar panel
      │ resource      │ from      │ type   │
      │ data.zip      │ meal plan │ file   │
      │ lorem.txt     │ meal plan │ file   │
      │ lorem-big.txt │ meal plan │ file   │
      │ meal plan     │           │ folder │
    ✖ failed
      locator.click: Timeout 30000ms exceeded.
      Call log:
        - waiting for locator('.oc-files-actions-show-details-trigger')

          at openForResource (/drone/src/webTestRunner/tests/e2e/support/objects/app-files/utils/sidebar.ts:18:70)
          at async Module.open (/drone/src/webTestRunner/tests/e2e/support/objects/app-files/utils/sidebar.ts:39:16)
          at async Module.downloadResources (/drone/src/webTestRunner/tests/e2e/support/objects/app-files/resource/actions.ts:507:17)
          at async Resource.download (/drone/src/webTestRunner/tests/e2e/support/objects/app-files/resource/index.ts:52:27)
          at async processDownload (/drone/src/webTestRunner/tests/e2e/cucumber/steps/ui/resources.ts:338:21)
          at async World.<anonymous> (/drone/src/webTestRunner/tests/e2e/cucumber/steps/ui/resources.ts:78:5)
```
